docker-compose.yml

```diff
@@ -1,7 +1,10 @@
 version: "3"
 
 services:
-### external modules
+
+  ### external modules
+  ####################################
```

Comment:
Antar linjen over er for å øke synligheten på overskriften/kommentaren. 
Er det noe mønster på antall tegn på denne ? (Dette er ikke et viktig spørsmål, ble bare nysgjerrig :) ).

Reply:
jeg bare holdte knappen ned for en stund... ikke noe macro eller autoformat.
så ikke garantert at det er samme lengte. 

En supersynlig standard kunne jo være 80 symbol? hele "linje"?

Reply:
Trenger ikke standard på alt, bare kjør på slik du har gjort :)